### DISCUSSION TITLE
Deeplink doesn't work on iOS

### DISCUSSION THREAD

**Discussion by SlvLabs** [WalletConnect/walletconnect-monorepo#3729] [Developer Support]
Hello,

we have migrated to v2, but deeplink doesn't for iOS, but it works at the same time for Android.

our project is Slavi wallet

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89833434/cb9efe17-1cd9-466a-b521-09a2eaf62107)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89833434/f6405d45-1395-4b09-98fe-37020270dbcb)

**Comment by glitch-txs** [comment]
hi could you share you project id or account email?

**Reply by SlvLabs** [reply]
Hi, sure. tech@slavi.io

**Reply by glitch-txs** [reply]
Thanks, which version of iOS is this?

**Reply by SlvLabs** [reply]
Latest version of iOS
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89833434/ff657b22-a8b1-4ae3-b2e5-8d831328b569)
here you can see results of the search using iOS

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89833434/6eb0d598-84f4-468c-ba85-9b313dda9ff6)
and here the same app with same deep link on Android

**Reply by glitch-txs** [reply]
Hi, the issue doesn't seem to come from our side, please check that you're providing the corrent link in the form
